std_msgs/Header header
aruco_multi_detect/ArUcoMarker[] markers
